CustomApps/lyrics-plus/Utils.js

```diff
@@ -54,4 +54,35 @@ class Utils {
     static capitalize(s) {
         return s.replace(/^(\w)/, ($1) => $1.toUpperCase());
     }
+
+    static isJapanese(lyrics) {
+        for (let lyric of lyrics)
+            if (
+                /[\u3000-\u303F]|[\u3040-\u309F]|[\u30A0-\u30FF]|[\uFF00-\uFFEF]|[\u4E00-\u9FAF]|[\u2605-\u2606]|[\u2190-\u2195]|\u203B/g.test(
+                    lyric.text
+                )
+            )
+                return true;
+        return false;
+    }
+
+    static rubyTextToReact(s) {
+        const react = Spicetify.React;
+
+        const rubyElems = s.split("<ruby>");
+        const reactChildren = [];
+
+        reactChildren.push(rubyElems[0]);
+
+        for (let i = 1; i < rubyElems.length; i++) {
+            let kanji = rubyElems[i].split("<rp>")[0];
+            let furigana = rubyElems[i].split("<rt>")[1].split("</rt>")[0];
```

Comment:
```suggestion
            const kanji = rubyElems[i].split("<rp>")[0];
            const furigana = rubyElems[i].split("<rt>")[1].split("</rt>")[0];
```

I didn't notice it at first glance and these variables aren't re-assigned so they can be assigned as `const`

Reply:
> That's all I noticed. About dict files, we'll see if we'll keep them in different repository instead of your own

Thats sound just about perfect!  Thanks for all of the suggestions, ill make sure to use const more often in the future xD